src/uct/ib/base/ib_iface.h

```diff
@@ -357,4 +359,66 @@ void uct_ib_iface_set_max_iov(uct_ib_iface_t *iface, size_t max_iov)
 }
 
 
+static UCS_F_ALWAYS_INLINE
+void uct_ib_iface_fill_ah_attr_raw(
+        uct_ib_iface_t *iface, uint16_t lid,
```

Comment:
need to indent according to code style (could pull void to prev line)

Reply:
will fix